presto-docs/src/main/sphinx/connector/accumulo.rst

```diff
@@ -0,0 +1,648 @@
+| accumulo.username                          | (none)           | Yes      | Accumulo user for Presto                                                                         |
++--------------------------------------------+------------------+----------+--------------------------------------------------------------------------------------------------+
+| accumulo.password                          | (none)           | Yes      | Accumulo password for user                                                                       |
++--------------------------------------------+------------------+----------+--------------------------------------------------------------------------------------------------+
+| accumulo.zookeeper.metadata.root           | /presto-accumulo | No       | Root znode for storing metadata. Only relevant if using default Metadata Manager                 |
++--------------------------------------------+------------------+----------+--------------------------------------------------------------------------------------------------+
+| accumulo.metadata.manager.class            | default          | No       | Fully qualified classname for the MetadataManager class. Default is the ZooKeeperMetadataManager |
++--------------------------------------------+------------------+----------+--------------------------------------------------------------------------------------------------+
+| accumulo.cardinality.cache.size            | 100000           | No       | Sets the size of the index cardinality cache                                                     |
++--------------------------------------------+------------------+----------+--------------------------------------------------------------------------------------------------+
+| accumulo.cardinality.cache.expire.duration | 5m               | No       | Sets the expiration duration of the cardinality cache.                                           |
++--------------------------------------------+------------------+----------+--------------------------------------------------------------------------------------------------+
+
+Unsupported Features
+~~~~~~~~~~~~~~~~~~~~
+
+Of the available Presto DDL/DML statements and features, the Accumulo connector does **not** support:
+
+- **Adding columns via ALTER TABLE**: While you cannot add columns via SQL, you can using a tool.
+  See the below section on `Adding Columns <#adding-columns>`__ for more.
+- **DELETE**: Deleting rows are not yet implemented for the connector
+
+Usage
+~~~~~
+
+Simply begin using SQL to create a new table in Accumulo to begin
+working with data. By default, the first column of the table definition
+is set to the Accumulo row ID. This should be the primary key of your
+table, and keep in mind that any ``INSERT statements`` containing the same
+row ID is effectively an UPDATE as far as Accumulo is concerned, as any
```

Comment:
Is this how Accumulo works in general? It seems to me that this should fail the INSERT, and the user should have to issue a DELETE first